Homework_5.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>The * means import all from that library (into the current namespace)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>